src/components/ResultsTable.tsx

```diff
@@ -26,15 +26,14 @@ export function ResultsTableWidget (props: ResultsTableProps) {
                         if (!result.Latitude || !result.Longitude) {
                             propertyAddress = `${result.Deal_Name} - Geocordinates N/A, cannot display on map.`
                         }
-
                         const ownerData = result.owner_details.find((owner) => owner.Contact_Type === 'Owner')
                         const contactData = result.owner_details.find((owner) => owner.Contact_Type === 'Director')
                         return (
                             <tr key={result.id}>
                                 <td>{index + 1}</td>
                                 <td>{propertyAddress}</td>
                                 <td>{ownerData?.Name || ''}</td>
-                                <td>{contactData?.Name || 'contact is not found'}</td>
+                                <td>{contactData?.Name || 'Contact Is Not Found'}</td>
```

Comment:
Maybe '' instead. I don't think we need a message like Contact Is Not Found unless Chris requested it.

Reply:
Chris asked it to be changed